src/Microsoft.DotNet.XHarness.Common/Microsoft.DotNet.XHarness.Common.csproj

```diff
@@ -1,26 +1,16 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>$(NetMinimum);netstandard2.0</TargetFrameworks>
+    <TargetFramework>$(NetMinimum)</TargetFramework>
```

Comment:
Did you verify that netstandard2.0 isn't needed anymore?

Reply:
From what I can see, yes